modules/op-creating-pipeline-tasks.adoc

```diff
@@ -18,31 +18,36 @@ $ oc create -f https://raw.githubusercontent.com/openshift/pipelines-tutorial/{p
 
 . Use the `tkn task list` command to list the tasks you created:
 +
+[source,terminal]
 ----
 $ tkn task list
 ----
 +
-The output verifies that the `apply-manifests` and `update-deployment` `Task` resources were created:
+The output verifies that the `apply-manifests` and `update-deployment` task resources were created:
 +
+[source,terminal]
 ----
 NAME                DESCRIPTION   AGE
 apply-manifests                   1 minute ago
 update-deployment                 48 seconds ago
 ----
 
-. Use the `tkn clustertasks list` command to list the Operator-installed additional `ClusterTask` resources, for example --`buildah` and `s2i-python-3`:
+. Use the `tkn clustertasks list` command to list the Operator-installed additional cluster tasks, for example --`buildah` and `s2i-python-3`:
 +
 [NOTE]
 ====
-You must use a privileged pod container to run the `buildah` `ClusterTask` resource because it requires a privileged security context. To learn more about security context constraints (SCC) for pods, see the Additional resources section.
+. You must use a privileged pod container to run the `buildah` cluster task because it requires a privileged security context. To learn more about security context constraints (SCC) for pods, see the Additional resources section.
```

Comment:
@vdemeester is this still true?

Reply:
@ppitonak indeed, it's not true anymore, for the clustertask we ship.

Reply:
Thanks Pavol!!